### PR TITLE
Announcing the first elections

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

File: _posts/2016-08-27-elections.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+Management Committee (PMC).
+
+TL;DR The Committees members are elected by the community for 1 year. They take
+care of the Code of Conduct and its values, participate in strategic planning,
+amongst other things.
+
+In normal times, the PMC would elect an Election Officer to run the elections,
+as stated in the [Governance Document][gd]. However, to bootstrap the first
+elections, two members have volunteered to do that task:
+[Daniele "daenney" Sluijters][daenney] and [me][me]. I will take care of
```
Comment (rnelson0) [inline _posts/2016-08-27-elections.md]: Use your name/nick, not 'me', for clarity, please.

Comment (roidelapluie) [inline _posts/2016-08-27-elections.md]: fixed

**Comment by daenney** [comment]
I'll fix the one weird test that trips up for documents not existing yet in a different PR.

File: _posts/2016-08-28-elections.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Vox Pupuli is about to live its first elections. This blogpost will give you the
+context and how to become a candidate.
+
+> Vox Pupuli is a collective of Puppet module, tooling and documentation authors
+> all working together to ensure continued development on the code we maintain.
+
+(Source: [https://voxpupuli.org/](https://voxpupuli.org/))
+
+Vox Pupuli now represents 68 collaborators, and 86 repositories. In order to
```
Comment (nibalizer) [inline _posts/2016-08-28-elections.md]: According to the website(which pulls from the github api) we have 93 Repositories.

I'd also say '68 member contributors'' and dozens more collaborators to include the folks that work with us but aren't members on the github org.

File: _posts/2016-08-28-elections.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+You can find more information [here][e], including how to become a candidate.
+Detailed information will be spread soon of this website too. Here is the
+timeline:
+
+* Before *September 17 2016 23:59UTC*: Accepting candidatures
+* Between *September 18 2016 12:00UTC* and *September 28 2016 12:00UTC*: Vote
+
+Please enjoy the party, and let's gather as much participation as possible
+during the whole election process!
```
Comment (nibalizer) [inline _posts/2016-08-28-elections.md]: I'd like to see a note about 'information on who can vote and how to vote will come soon'

File: _posts/2016-08-28-elections.md (hunk 2 above)
Comment (roidelapluie) [inline _posts/2016-08-28-elections.md]: Then we need to fix the homepage of https://voxpupuli.org/

File: _posts/2016-08-28-elections.md (hunk 3 above)
Comment (roidelapluie) [inline _posts/2016-08-28-elections.md]: its there: "+Detailed information will be spread soon of this website too. Here is the
+timeline:"

File: _posts/2016-08-28-elections.md (hunk 2 above)
Comment (roidelapluie) [inline _posts/2016-08-28-elections.md]: I see 86
repos

**Pull request merged**